### PR TITLE
Minimum element in a stack in constant time in C

### PR DESCRIPTION
**Pull request by sumitgsh**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
YES
### Description

(Write your answer here.)
Find a minimum element in a stack in constant time
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#2972

### CODE REVIEW

**Comment by sumitgsh** [comment]
@ankitaggarwal23 @bhav09 @anushkrishnav  can u please verify my PR and merge

File: C/ds/MinimumElemetInStack.c [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  11
+  5
+  7
+  2
+  -5
+
+  -5 is popped from the stack:
+  Smallest element in the Current Stack: 2
+
+  Time complexity: O(1) for finding the minimum
```
Comment (bhav09) [inline C/ds/MinimumElemetInStack.c]: Can you write the time complexity above Sample I/O. 
Rest everything seems fine.

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

File: C/ds/MinimumElemetInStack.c (hunk 1 above)
Comment (sumitgsh) [reply C/ds/MinimumElemetInStack.c]: @bhav09 can u please verify and merge

File: C/ds/MinimumElemetInStack.c [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    of the Original Stack.
+
+    Every push to the primary STACK, compare the current value
+    with the TOS of the temporary Stack and IF the current value is less
+    than the TOS push the current value to the temporary STACK,such that
+    the minimum element is always at the top of the STACK
+
+*/
+
+#include <bits/stdc++.h>
```
Comment (ankitaggarwal23) [inline C/ds/MinimumElemetInStack.c]: Remove `#include <bits/stdc++.h>`

File: C/ds/MinimumElemetInStack.c [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  11
+  5
+  7
+  2
+  -5
+
+  -5 is popped from the stack:
+  Smallest element in the Current Stack: 2
+  
+ */
```
Comment (ankitaggarwal23) [inline C/ds/MinimumElemetInStack.c:144]: Capture

Kindly cross check your output.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C/ds/MinimumElemetInStack.c (hunk 2 above)
Comment (sumitgsh) [reply C/ds/MinimumElemetInStack.c]: @ankitaggarwal23 marked and updated

File: C/ds/MinimumElemetInStack.c (hunk 3 above)
Comment (sumitgsh) [reply C/ds/MinimumElemetInStack.c:144]: @ankitaggarwal23  made the change,can u please review

**Review by ankitaggarwal23 — APPROVED** [review]

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**